src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_lord_jaraxxus.cpp

```diff
@@ -21,6 +21,7 @@
 #include "SpellScript.h"
 #include "Player.h"
 #include "trial_of_the_crusader.h"
+#include <math.h>
```

Comment:
Using same thing in https://github.com/TrinityCore/TrinityCore/commit/b11545800190795de5573fb917e81f61b774427b#diff-68036f4e322884df2c7c2c4819780395R22

here: https://github.com/TrinityCore/TrinityCore/blob/9bbca372b6395ebf97d898f018459cc88f3c3a38/src/server/scripts/EasternKingdoms/SunwellPlateau/boss_kiljaeden.cpp#L30

and here: https://github.com/TrinityCore/TrinityCore/blob/a56a402190c885e51038df55654875cc9e875c03/src/server/game/Entities/Player/PlayerTaxi.cpp#L5

is really a problem?